sass/_variables.scss

```diff
@@ -17,11 +17,15 @@ $white-transluant:       rgba($white, .8) !default;
 // Brand colors
 // -------------------------
 
-$brand-primary:         #f4d430 !default;
+// Could also be #3EA0DA, depending on HBP identity renewal
+$brand-primary:         #82BBC4 !default; // was F4D430 in v1
```

Comment:
Rtfm ? :D Well almost. In my commit message concerning this line, I mention changing the colours but all buttons keep a yellow colour. Most if not all pages will remain unaffected by this.